src/main/java/api/JsonCollegeScorecardDB.java

```diff
@@ -98,7 +98,7 @@ public JSONObject zipcodeQuery(String zipcode, String radius) {
         // query a radius around a zip code for institutions
         // NOTE: Make sure to add "mi" at the end of radius!
         // create the query url
-        String url = API_ADDRESS + "schools.json?" + "zip=" + zipcode + "&" + "distance=" + radius + "&fields=id,school.name,2013.student.size" + "&fields=id,school.name,school.state,school.city,admissions.admission_rate.overall,cost.tuition.in_state,cost.tuition.out_of_state,admissions.sat_scores.average.overall,admissions.act_scores.midpoint.cumulative,school.school_url" + "&api_key=" + API_TOKEN;
+        String url = API_ADDRESS + "schools.json?" + "zip=" + zipcode + "&" + "distance=" + radius + "&fields=id,school.name,2013.student.size" + "&fields=id,school.name,school.state,school.city,2018.admissions.admission_rate.overall,2018.cost.tuition.in_state,2018.cost.tuition.out_of_state,2018.admissions.sat_scores.average.overall,2018.admissions.act_scores.midpoint.cumulative,school.school_url" + "&api_key=" + API_TOKEN;
```

Comment:
good change specifying year here. If you find a lot of null, consider switching to an even older year like 2013